From c5cb4e1a0339844ae3f55ff1dc4a716c28012f05 Mon Sep 17 00:00:00 2001
From: Paul Crawford <psc@sat.dundee.ac.uk>
Date: Tue, 28 Jun 2016 18:08:48 +0100
Subject: [PATCH 01/10] Include linux/param.h for EXEC_PAGESIZE definition

Musl does not include linux/param.h whereas glibc does, so it fails
to build on musl. Patch supplied by Khem Raj <raj.khem@gmail.com>
---
 src/watchdog.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/watchdog.c b/src/watchdog.c
index acf6450..486384a 100644
--- a/src/watchdog.c
+++ b/src/watchdog.c
@@ -26,6 +26,9 @@
 #include <sys/param.h>		/* For EXEC_PAGESIZE */
 #include <linux/oom.h>
 #include <linux/watchdog.h>
+#ifdef __linux__
+#include <linux/param.h>
+#endif
 #include <string.h>
 
 #include <libgen.h>
-- 
2.20.1

